### PR TITLE
Add Sphinx documentation

### PR DESCRIPTION
**Pull request by jungshadow**
This is related to #48 (which I reopened). The documentation isn't entirely finished, but I'd rather get something in the repository so this can be more of a collective effort (Aside: there's a lot of documentation and I've been dedicating a lot more time to this than I actually have available). I'll keep plugging away on it, but this gives us a starting point.

### CODE REVIEW

**Comment by jungshadow** [comment]
Also, apologies for all of the superfluous deleted files after deleting the `_build` folder. I didn't want to squash the commits since that would destroy @jdmgoogle's contributions.

**Comment by jungshadow** [comment]
Can someone take a quick look at this? Thanks! /cc @jdmgoogle @jktomer @pstenbjorn @cjerdonek

**Comment by pstenbjorn** [comment]
@jungshadow Content looks good to me.  One style question, is there a reason you haven't embedded links to enums where enums are referenced in the elements?  If there isn't a reason, it might be helpful.

File: docs/xml/elements/ballot_measure_contest.rst [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+|                  |<internationalized_text>`                          |             |          |referendum that is on the ballot, below   |invalid or not present, |
+|                  |                                                   |             |          |the title, but above the text.            |the implementation is   |
+|                  |                                                   |             |          |                                          |required to ignore it.  |
+|                  |                                                   |             |          |                                          |                        |
+|                  |                                                   |             |          |                                          |                        |
++------------------+---------------------------------------------------+-------------+----------+------------------------------------------+------------------------+
+| Type             |:doc:`BallotMeasureType                            | Optional    | Single   |Specifies the particular type of ballot   |If the element is       |
```
Comment (jungshadow) [inline docs/xml/elements/ballot_measure_contest.rst]: @pstenbjorn I might not have done it everywhere, but I tried link to the enumerations. If possible, I'd like to get this in and start making the fixes in subsequent pull requests.

**Comment by pstenbjorn** [comment]
@jungshadow  then LGTM

**Comment by jktomer** [comment]
What's the best way for me to actually view the generated documentation?

**Comment by jktomer** [comment]
LGTM

**Comment by jungshadow** [comment]
EXCELLENT!

**Pull request merged**